src/index.ts

```diff
@@ -0,0 +1,15 @@
+import { initServer } from "./app";
+
+import * as dotenv from "dotenv";
+
+dotenv.config();
+
+let PORT: number;
+// Retrieve the listening port from the environment variables
+PORT = Number(process.env.PORT);
+if (!PORT) {
```

Comment:
You can use a the or operator to assign a default value.

`const port: number = +(process.env.PORT || 3000)`